src/test/Components/Blueprints/Blueprints.test.js

```diff
@@ -39,4 +43,17 @@ describe('Blueprints', () => {
     renderWithReduxRouter('', {});
     await screen.findByText('No blueprints yet');
   });
+  test('renders blueprint composes', async () => {
```

Comment:
Once we've updated the fixture so that 'Dark Chocolate' has two images associated with it and 'Milk Chocolate' has none, we can also update this test to reflect that. It should check for both images associated with 'Dark Chocolate', and then either in this test or another test check that the empty images state is displayed for 'Milk Chocolate'.